addons/mail/static/tests/new/crosstab/crosstab_tests.js

```diff
@@ -44,3 +46,20 @@ QUnit.test("Delete starred message updates counter", async function (assert) {
     );
     assert.containsNone(tab2.target, "button:contains(Starred1)");
 });
+
+QUnit.test("Thread rename", async function (assert) {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["mail.channel"].create({
+        name: "General",
+    });
+    const tab1 = await start({ asTab: true });
+    const tab2 = await start({ asTab: true });
+    await tab1.openDiscuss(channelId);
+    await tab2.openDiscuss(channelId);
+    await tab1.insertText(".o-mail-discuss-thread-name", "Sales", true);
```

Comment:
Boolean as positional argument is not super clear. I changed so it's `{ replace: true }` instead